ed25519/shamirutil/misc.go

```diff
@@ -0,0 +1,26 @@
+package shamirutil
```

Comment:
This file (misc.go) probably doesn't need to be copied over. All of the functions in this file are exactly the same.

Reply:
Removed the misc.go file. Using the misc code from secp256k1.